tests/x11regressions/tracker/tracker_searchall.pm

```diff
@@ -18,15 +18,6 @@ sub run() {
     sleep 2;
     wait_idle;
     assert_screen 'tracker-needle-launched';
-    send_key "tab";
```

Comment:
I guess we now reached the point where it makes sense to create maintenance branches for the tests and needles repo.